index.html

```diff
@@ -71,7 +72,220 @@ <h1>I’m Sylvester,
           </li>
         </ul>
       </div>
-    </section>        
+    </section>
+    <section class="card">
+      <div class="card-img">
+        <img src="images/portfolio-tonic.png" alt="Tonic Portfolio">
+      </div>
+      <div class="card-head">
+        <h1>Tonic</h1>
+      </div>
+      <div class="card-info">
+        <span class="canopy">CANOPY</span>
+        <i class="fas fa-circle fa-xs"></i>
+        <span class="head-item">Back End Dev</span>
+        <i class="fas fa-circle fa-xs"></i>
+        <span class="head-item">2015</span>
+      </div>
+      <div class="card-para">
+        <p>
+          A daily selection of privately personalized reads; no accounts
+          or sign-ups required.
+        </p>
+      </div>
+      <div class="card-lang">
+        <ul>
+          <li class="html">html</li>
+          <li class="css">css</li>
+          <li class="js">javaScript</li>          
+        </ul>
+      </div>
+      <div class="seeport">
+        <span>
+          <a href="#">See Project</a>
+        </span>
```

Comment:
This issue is addressed. Nice work! :clap: . But it would have been even better if you have replaced `span` with `button` tag. Anyways, I am ok with it.